pep/app/services/pep-session.ts

```diff
@@ -37,7 +37,6 @@ export default class PepSessionService extends SessionService.extend({
     setUnauthenticatedSession(sessionData: PepSecureAuthenticatedData) {
         const resultString = JSON.stringify(sessionData);
         this.cookies.write(UNAUTHENTICATED_SESSION_COOKIE_NAME, resultString, {
-            domain: ENV.cookieDomain,
             secure: this.isCookieSecure,
             sameSite: ENV.cookieSameSite,
             expires: new Date('2525-01-01') // never!!!
```

Comment:
I think it would be beneficial to bring this, and the secure values over into the cookie service since they are used in multiple places. So it would end up looking like

```
this.cookies.write(UNAUTHENTICATED_SESSION_COOKIE_NAME, resultString, {
    secure: this.cookies.isCookieSecure,
    expires: this.cookie.expirationDate
...
```

Reply:
we dont have our own `cookies` service defined, its just using the addon's service directly.  i guess i could create one, it would need to be named differently due to the .js/.ts collision issue, but id rather save that for a clean up pass in my next branch so as not to hold these fixes up.

Reply:
Maybe a half fix is just moving the expires into a constant and creating a global helper for isCookieSecure

Reply:
yeah i can do that. actually, `cookieSecure` should just really be converted to boolean directly when its set in environment.js:
```js
cookieSecure: Number(process.env.COOKIE_SECURE) === 1,
```